docs/source/configurator.rst

```diff
@@ -99,6 +99,7 @@ Node type dependent options for [nodeN] : ::
    ├─sequence_identifier : sequence identifier, default "re-sequencer"
    ├─segment_length : duration of each output segment in seconds, default 2
    ├─begin_output : ["immediate" (default) | {begin time} ] the time at which the first output segment should begin.
+   ├─init_document : File path to a document used to set initial parameters and afterwards start issuing documents (instead of awaiting the first received document), default None
```

Comment:
It'd be good to add a section of documentation explaining how the init document is used and what constraints there are for it. For example, what happens if the init document has a different sequence id to the sequence we are connecting to? Does that mean that the expected incoming documents cause an error? (I expect it does but have not checked yet)

Reply:
Adding more explanation makes sense. What location within the docs would you prefer for such section? Ddocuments with a sequence ID different to the init document's one are discarded, raising an `IncompatibleSequenceError` with `ERR_DOCUMENT_NOT_COMPATIBLE`.

Reply:
Oh dear, I've found #447 as a result of this. We never documented the resequencer properly! We should have added it to `docs/source/scripts_and_their_functions.rst`. I'll open a quick pull request for that issue; if you could add some extra text to that pull request, that would be helpful. Otherwise I think this is now good to merge.

Reply:
OK I've opened #510 to address #447, so if you review that and we merge it @spoeschel then you can merge that pull request here too and then add the documentation to the new section introduced there, or we can do it the other way round, merge this and then add to the Resequencer documentation. I don't mind which.

Reply:
In this case I would prefer to first have this MR being merged. I then would add a commit to #515, adding documentation for this MR, and having that MR being merged, too. This saves us an additional issue/MR process.

Reply:
Sorry I did this the other way around. I lost track a bit!

Reply:
No problem!